conan/cli/commands/config.py

```diff
@@ -24,8 +24,11 @@ def config_install(conan_api, parser, subparser, *args):
                            help="git repository, local file or folder or zip file (local or "
                                 "http) where the configuration is stored")
 
-    subparser.add_argument("--verify-ssl", nargs="?", default="True",
-                           help='Verify SSL connection when downloading file')
+    ssl_subgroup = subparser.add_mutually_exclusive_group()
+    ssl_subgroup.add_argument("--verify-ssl", nargs="?", default="True",
+                              help='Verify SSL connection when downloading file')
+    ssl_subgroup.add_argument("--insecure", action="store_true", default=None,
```

Comment:
Document it is equivalent to --verify-ssl=False

Maybe use https://docs.python.org/3/library/argparse.html#dest?